src/components/NewVariantTable.tsx

```diff
@@ -46,7 +47,9 @@ export const NewVariantTable = ({ country, year, week, onVariantSelect }: Props)
             <tbody>
               {data.map(d => (
                 <tr key={d.variant.mutations.join(',')}>
-                  <td style={{ maxWidth: '400px', lineBreak: 'auto' }}>{d.variant.mutations.join(', ')}</td>
+                  <td style={{ maxWidth: '400px', lineBreak: 'auto' }}>
+                    <MutationList mutations={d.variant.mutations} />
```

Comment:
My browser is not happy rendering all of these React components and DOM elements here in this huge table. Can we keep `d.variant.mutations.join(', ')` within this table?

Reply:
To see the issues: load all of the B1.1.7 samples, resize the width of your page and then try scrolling.